.github/actions/upload-alerts/action.yml

```diff
@@ -22,15 +22,14 @@ runs:
 
     - name: Run Script to Validate Alerts
       shell: bash
-      if: github.event_name != 'pull_request'
       run: | 
-        python tools/scripts/validate_alerts.py --alerts "${{ inputs.alerts }}"
+        python _test-infra/tools/scripts/validate_alerts.py --alerts '${{ inputs.alerts }}'
 
     - name: Run Script to Upload Alerts
```

Comment:
Do you need to pass `ROCKSET_API_KEY` as an env variable here?  I see that this is used in the script but is not set anywhere in the GH action

Reply:
fortunately, no. as long as its passed in the action.